modules/gltf/gltf_document.cpp

```diff
@@ -2756,6 +2785,7 @@ Error GLTFDocument::_parse_meshes(Ref<GLTFState> p_state) {
 
 		Array primitives = d["primitives"];
 		const Dictionary &extras = d.has("extras") ? (Dictionary)d["extras"] : Dictionary();
+		_attach_extras_to_meta(extras, mesh);
```

Comment:
This will cause all imported meshes to have extras on it, even if the GLTF file did not have any. We should wrap this line in a check for `d.has("extras")`.

Reply:
As the existing code always instantiates `Dictionary &extras` the function `_attach_extras_to_meta` does check if the `extras` dictionary is non empty, so I thought it would be superfluous to do it here. I can change it, but it feels like it should be handled there.

Reply:
Ah, you're right. Never mind then.